packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -526,10 +543,30 @@ export function TransferPanelMain({
         }
       },
       to: {
-        disabled: true,
-        options: [to],
+        options: [l1.network, ...options],
         value: to,
-        onChange: () => {}
+        onChange: async network => {
+          // Selecting the same chain or L1 network
+          if (to.chainID === network.chainID || isL1Network(network)) {
+            return
+          }
+
+          // Destination network is L2, connect to L1
```

Comment:
Main issue with this logic is that users can now see both One & Nova on the dropdowns
so they'd try to use our UI to bridge from One to Nova
but choosing One would by default set the destination to Mainnet
and if they change the destination dropdown to Nova, the `from` dropdown would be switched to `Mainnet`
so the user would be very angry (╯°□°)╯( ┻━┻

Reply:
Discussed possible approaches with @spsjvc as well and we'd like to go with all possible dropdowns for now and have logic to disallow one<>nova bridging. In the near future we will allow one<>nova, and I can see UX concerns here.

I'm happy to discuss it further because I have similar thoughts on it